### PR TITLE
[RW-5968][risk=no] Add notes on running genomics tools on unsorted vcf input

### PR DESCRIPTION
**Pull request by ericsong**
---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by als364 — APPROVED** [review]
Thank you for writing this up!

File: api/genomics/docs/unsorted_vcf.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Answering some questions on how our tools interact with unsorted VCF files
+
+### Does Plink require a VCF file to be sorted?
+No, but another flag `--make-bed` needs to be added to the binary file creation step.
```
Comment (calbach) [inline api/genomics/docs/unsorted_vcf.md:4]: One thing to understand here is whether we are just pushing the sorting burden to later in the flow. Presumably `--make-bed` is actually sorting the variants, and therefore needs to do the same work as the input VCF.

File: api/genomics/docs/unsorted_vcf.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+`plink --vcf-half-call m --const-fid 0 --vcf unsorted.vcf --out cohort --make-bed` ->  
+`plink --vcf-half-call m --const-fid 0 --vcf unsorted.vcf --out cohort --make-bed`
```
Comment (calbach) [inline api/genomics/docs/unsorted_vcf.md:6]: What happens if you don't use `--make-bed` ? Does it error out?

(also, nice flag name)

Comment (ericsong) [reply api/genomics/docs/unsorted_vcf.md:6]: yeah. it will fail without the flag

File: api/genomics/docs/unsorted_vcf.md (hunk 1 above)
Comment (ericsong) [reply api/genomics/docs/unsorted_vcf.md:4]: I couldn't find documentation that explicitly stated this but that's almost certainly the case.

**Pull request merged**